src/utils/CompilerLocationUtils.fs

```diff
@@ -99,10 +99,10 @@ module internal FSharpEnvironment =
         Option.ofString
             (try
                 let key = RegistryKey.OpenBaseKey(RegistryHive.LocalMachine, RegistryView.Registry32)
-                if key = null then null
+                if isNull key then null
```

Comment:
@forki It's interesting that other places you cleaned up if else's to match with.  And here you didn't.  How do you choose which to use?